gr-zeromq/python/zeromq/qa_zeromq_pubsub.py

```diff
@@ -48,5 +48,28 @@ def test_001 (self):
         self.tb.wait()
         self.assertFloatTuplesAlmostEqual(sink.data(), src_data)
 
+    def test_002(self):
+        # Test getting the port returns a port we explicitly set endpoint to
+        zeromq_pub_sink = zeromq.pub_sink(gr.sizeof_float, 1, "tcp://127.0.0.1:5556", 0)
+        self.assertEqual(5556, zeromq_pub_sink.port())
```

Comment:
One issue that might arise from hard coding the port number here is if there are other tests that do the same, and run in parallel. Not sure I know the answer yet but I wanted to bring this up.